### PR TITLE
ABZ Case studies publications....

### PR DESCRIPTION
**Pull request by nksingh-n7**
This PR introduces the following changes:

abz14, abz16, abz18 and abz20 case studies publications are updated. 
**"Please contact authors"** is added and the required authors emails are provided.

### CODE REVIEW

File: .github/src/content/publication/abz14/arcaini2014abz.md [hunk 1]
```diff
@@ -85,12 +85,12 @@ pages="36--51",
 abstract="The paper presents an Abstract State Machine (ASM) specification of the Landing Gear System case study, and shows how the ASMETA framework can be used to support the modeling and analysis (validation and verification) activities for developing a rigorous and correct model in terms of ASMs. We exploit the two fundamental concepts of the ASM method, i.e., the notion of ground model and the refinement principle, and we achieve model development and model analysis by the combined use of formal methods for specification and for verification.",
 isbn="978-3-319-07512-9"
 }
-~~~
+```
```
Comment (ppaulweber) [inline .github/src/content/publication/abz14/arcaini2014abz.md:88]: @nksingh-n7 both syntax styles are valid for a markdown source code block

File: .github/src/content/publication/abz14/arcaini2014abz2.md [hunk 2]
```diff
@@ -2,56 +2,56 @@
 title: "Offline Model-based Testing and Runtime Monitoring of the Sensor Voting Module"
 
 authors:
-- Paolo Arcaini
-- Angelo Gargantini
-- Elvinia Riccobene
+  - Paolo Arcaini
+  - Angelo Gargantini
+  - Elvinia Riccobene
```
Comment (ppaulweber) [inline .github/src/content/publication/abz14/arcaini2014abz2.md:7]: @nksingh-n7 could it be that you used a editor which did automatic reformatting of all the bullet lists?

**Review by ppaulweber — COMMENTED** [review]
LGTM :+1: Please just try to keep the change set small, and content related changes and format related changes in two different PRs, because the PR delta was mainly format related whereas the context of the PR is content related.

**Review by ppaulweber — APPROVED** [review]

File: .github/src/content/publication/abz14/arcaini2014abz2.md (hunk 2 above)
Comment (nksingh-n7) [reply .github/src/content/publication/abz14/arcaini2014abz2.md:7]: I had used vscode for working on the website. I think the editor of vscode did automatic reformatting of all the bullet lists.

File: .github/src/content/publication/abz14/arcaini2014abz.md (hunk 1 above)
Comment (nksingh-n7) [reply .github/src/content/publication/abz14/arcaini2014abz.md:88]: I think vscode had automatically generated this symbol for the markdown....

File: .github/src/content/publication/abz14/arcaini2014abz2.md (hunk 2 above)
Comment (ppaulweber) [reply .github/src/content/publication/abz14/arcaini2014abz2.md:7]: I thought so, but it is not a problem, I'll try to comply with this format in the future with my changes and then we can keep the change set small.

**Pull request merged**